src/app/modules/ix-forms/components/ix-input/ix-input.component.spec.ts

```diff
@@ -27,16 +29,22 @@ describe('IxInputComponent', () => {
   });
 
   describe('rendering', () => {
-    it('renders a hint when it is provided', () => {
-      spectator.setInput('hint', 'Capital letters only');
+    it('renders a label and passes properties to it', () => {
+      spectator.setInput('label', 'New Password');
+      spectator.setInput('required', true);
+      spectator.setInput('tooltip', 'Minimum length is 8 characters.');
 
-      expect(spectator.query('mat-hint')).toHaveText('Capital letters only');
+      const label = spectator.query(IxLabelComponent);
```

Comment:
This kind of testing feels a bit like cheating. We're just checking variable values. Not what's on the actual dom. There can be failure points between variables and dom still. Sort of like doing `this = that; if (this == that) ... `

Reply:
Not really, `spectator.query(IxLabelComponent)` will return nothing if component is not in the dom.
So here we check that ix-label is there and received correct inputs.
And we make sure that ix-label renders label correctly in its test file.